docs/development_suite/set-up-infrastructure/enhanced-project-workflow.md

```diff
@@ -162,6 +165,19 @@ With this feature Users will be able to not only review the configurations but a
 
 With the first release of this new Workflow, existing Projects will require a migration operation to be performed. With this improvement we ought to provide further assistance in the migration process to make it as easy and smooth as possible.
 
+## Missing features
+
+### Secret Variables Management
```

Comment:
This is not a missing _feature_

interpolating secrets creates vulnerable deployment by exposing secret values in plain text in deployment/configmap configurations.

The enhanced workflow is preventing you from doing such errors, I'd put it more in terms of: "if you are interpolating secrets fix the issue before migrating to the enhanced workflow